packages/spec-lib/package.json

```diff
@@ -17,7 +17,10 @@
   "main": "dist/index.js",
   "tspMain": "lib/lib.tsp",
   "exports": {
-    ".": "./dist/src/index.js"
+    ".": {
```

Comment:
Surprised this didn't have a `types` field before. Maybe needs one for `./dist/src/index.d.ts`?

Actually looks like the `main` export also should be `./dist/src/index.js` (I guess node.js prefers `exports` over `main`)

Reply:
typescript will infer the typez auto from import or default

Reply:
but yeah made an error copying here